ThunderRequestTests/DownloadTests.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  DownloadTests.swift
+//  ThunderRequestTests
+//
+//  Created by Simon Mitchell on 14/12/2018.
+//  Copyright © 2018 threesidedcube. All rights reserved.
+//
+
+import UIKit
+import XCTest
+@testable import ThunderRequest
+
+class DownloadTests: XCTestCase {
+    
+    let requestBaseURL = URL(string: "https://via.placeholder.com/")!
+    
+    func testDownloadSavesToDisk() {
+        
+        let requestController = RequestController(baseURL: requestBaseURL)
+        let finishExpectation = expectation(description: "App should correctly download body from server")
+        
+        requestController.download("500", progress: nil) { (response, url, error) in
```

Comment:
Is this making a network request? If so, this could fail if we're on the office WiFi / the service goes down.

Reply:
It is, it's an integration test :) It 100% could fail, but not sure what else we can do about it? :/